app/data/toys.json

```diff
@@ -127,6 +137,21 @@
             "icon": "inv_misc_trinket6oog_tablet1",
             "itemId": 170155,
             "name": "Carved Ankoan Charm"
+          },
+          {
+            "icon": "inv_knife_1h_grimbatolraid_d_03",
+            "itemId": 173951,
+            "name": "N'lyeth, Sliver of N'Zoth"
```

Comment:
Debatable but I'd rather have this in a BFA > World PvP subcategory

Reply:
Fixed.